### ISSUE TITLE
Review Distributions.jl

### ISSUE THREAD

**Issue by gecrooks** [gecrooks/fieldguide#4]
Review the distributions implemented (or planned to be implemented) in Distributions.jl (https://github.com/JuliaStats/Distributions.jl), and look for distributions missing from the field guide. See issue #1

https://github.com/JuliaStats/Distributions.jl/issues/124

https://github.com/taqtiqa-mark/Distributions.jl/blob/563ea34b42e94277baa918ccc9610a3d5b6b838d/docs/src/univariate-list.md

**Comment by azev77** [comment]
Hi @gecrooks, @MAzarbad submitted a PR adding [Generalized Gamma](https://github.com/JuliaStats/Distributions.jl/pull/1166). 
I suggested possibly adding a location parameter to get the Amoroso distribution. 
Have you seen any codes/packages that fit the Amoroso distribution?

**Comment by gecrooks** [comment]
No, I don't recall seeing code to fit the Amoroso distribution. But fitting distributions is a problem that I deliberately avoided talking about in the Field Guide, since it is a large and amorphous subject, and hard to summarize.
